### ISSUE TITLE
release system is down?

### ISSUE THREAD

**Issue by Jeffwan** [kubeflow/testing#812]
Seems CD system didn't pick changes from repos kubeflow/kubeflow for a while. 

See centraldashboard latest image tag was built this Aug. 
https://console.cloud.google.com/gcr/images/kubeflow-images-public/GLOBAL/centraldashboard?gcrImageListsize=30

Change list 
https://github.com/kubeflow/kubeflow/commits/master/components/centraldashboard

I think https://github.com/kubeflow/kubeflow/commit/8d36a3887b54eec2e866d0a6df7e2b1c3c2b7a7c#diff-11c80890c5420bf75d645500ab81851788d9651eb6725194c9b5487e1e5827ee this change should be picked by CD.

I did some check and here's the latest autoPR 
https://github.com/kubeflow/manifests/pull/1592

I filed https://github.com/kubeflow/testing/pull/804 and seems it doesn't build new image tags from new branch

https://github.com/kubeflow/kubeflow/issues/5371

**Comment by Bobgy** [comment]
I'll take a look. Please take your time blocked on the issue thinking about each WG's own release infra, because 20% contributions will always mean not enough bandwidth to maintain this stuff.

**Comment by Bobgy** [comment]
Haven't looked through, here's documentation for the release infra and it means postsubmit tests taking a key role in this.
So I wonder if the broken release system related to test infra changes: https://github.com/kubeflow/kubeflow/blob/master/docs_dev/releasing.md

/cc @PatrickXYS

**Comment by PatrickXYS** [comment]
I'm not familiar with release infra which is set up previously, but I can take a look to see if we can root-cause.

We have release-infra story alongside with test-infra, they're on our roadmap, will be discussing with WGs very soon

**Comment by Jeffwan** [comment]
@Bobgy No problem. just a heads up. @thesuperzapper will raise this in community meeting. We can have some discussion later. Asking WG own release infra makes sense as well, however, we might miss some repos. Let's see. If AWS can sponsor the release infra, that is great as well

**Comment by Bobgy** [comment]
I suspect this is related https://github.com/kubeflow/testing/issues/750, so I'm working on https://github.com/kubeflow/testing/pull/813.

It will take a while until I can clean up all the old tekton pipelineruns from the kf-releasing cluster.

**Comment by jlewi** [comment]
As @Bobgy says at this point the WG leads should be responsible for building and releasing their own images.

**Comment by Jeffwan** [comment]
1.2 release doesn't have dependency on this issue. release infra is a separate story. As we discussed, this should be owned by  each WG. For user who like to work on this part for your own release infra, check https://github.com/kubeflow/testing/tree/master/apps-cd
